Archive/Imo/Imo1997Q3.lean

```diff
@@ -0,0 +1,128 @@
+/-
+Copyright (c) 2025 Jeremy Tan. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jeremy Tan
+-/
+import Mathlib.Algebra.BigOperators.Ring.Finset
+import Mathlib.Algebra.Group.Submonoid.Membership
+import Mathlib.Data.Real.Sign
```

Comment:
`Mathlib.Data.Real.Sign` ought to be deprecated (replace lemmas with ones for `SignType.sign` if it doesn't already have corresponding lemmas) rather than adding a new use of it; see #7842. Please use `SignType.sign` instead of `Real.sign`.